pkg/harvester/models/kubevirt.io.virtualmachine.js

```diff
@@ -170,13 +170,13 @@ export default class VirtVm extends HarvesterResource {
       },
       {
         action:  'migrateVM',
-        enabled: !!this.actions?.migrate,
+        enabled: !!this.actions?.migrate && !this.vmi?.vmimResource,
```

Comment:
The `this.vmi?.vmimResource` is an empty array before VM migration, after VM migrated the `this.vmi.vmiiResource` has steveObject (see screenshot), but it won't be clean up. That casuses the migrate button is disappered.

We should allow VM migrate to node A and then able tp migrate back to node B, right ? 

![Screenshot_20240724_114819](https://github.com/user-attachments/assets/b58cce68-1827-4d05-9283-e279fa8de24f)
![Screenshot_20240724_120215-1](https://github.com/user-attachments/assets/5023b0b2-b970-4b3a-a31e-8094a7027de7)


IMO, the better fix is from backend to update action list and frontend just honors the action list to show corresponding action button.